src/main/java/com/annapolisworks/Operator.java

```diff
@@ -0,0 +1,48 @@
+package com.annapolisworks;
+
+public enum Operator {
+    PLUS('+', 1),
+    MINUS('-', 1),
+    MULTIPLY('*', 2),
+    DIVIDE('/', 2),
+    EXPONENT('^', 3);
+
+    private char mOper;
+    private int mPriority;
+
+    Operator(char operator, int priority) {
```

Comment:
You're right. Enums are the exception. Enum constructors are always private. :heavy_check_mark: